scripts/pre-install.sh

```diff
@@ -1,10 +1,16 @@
-#!/bin/sh
+#!/bin/bash
 
-# Copy existing configuration if pre-existing installation is found
-test -f /etc/opt/influxdb/influxdb.conf
-if [ $? -eq 0 ]; then
-    mkdir /etc/influxdb
-    cp -a /etc/opt/influxdb/* /etc/influxdb/
-fi
+if [[ -d /etc/opt/influxdb ]]; then
+    # Legacy configuration found
+    if [[ ! -d /etc/influxdb ]]; then
+	# New configuration does not exist, move legacy configuration to new location
+	echo -e "Please note, InfluxDB's configuration is now located at '/etc/influxdb' (previously '/etc/opt/influxdb')."
+	mv -vn /etc/opt/influxdb /etc/influxdb
 
-exit
+	if [[ -f /etc/influxdb/influxdb.conf ]]; then
```

Comment:
Yes, that is just to make 100% sure we have a backup. I've encountered a few issues with rpm where it won't save it correctly because the configuration file changed locations. In those instances, it creates an `influxdb.conf.$(date)~` backup, but it's just a copy of the vanilla configuration (without any customizations).